### PR TITLE
Fix alluxio-fuse unmount to get the right pid

### PR DESCRIPTION
**Pull request by secfree**
### What changes are proposed in this pull request?

Fix alluxio-fuse unmount to get the right pid to kill.

### Why are the changes needed?

Currently even the "-f" option cannot kill the AlluxioFuse process, below is a demo

```
$ jps -m
42757 AlluxioFuse -m /mnt/projects -a /projects -o allow_other

$ integration/fuse/bin/alluxio-fuse unmount -f /mnt/projects
Successfully unmount fuse at /mnt/projects

$ jps -m
42757 AlluxioFuse -m /mnt/projects -a /projects -o allow_other
```

The cause is:
- https://github.com/Alluxio/alluxio/blob/master/integration/fuse/bin/alluxio-fuse#L101

    ```
      alluxio.fuse.AlluxioFuse \
      -m ${mount_point} -a ${mount_alluxio_path} ${mount_options}"
    ```

- https://github.com/Alluxio/alluxio/blob/master/integration/fuse/bin/alluxio-fuse#L205, `fuse_stat` outputs the `mount_alluxio_path` instead of `mount_point` here

### Does this PR introduce any user facing changes?

No

### CODE REVIEW

File: integration/fuse/bin/alluxio-fuse [hunk 1]
```diff
@@ -202,7 +202,7 @@ fuse_stat() {
   local fuse_info="$(ps aux | grep [A]lluxioFuse)"
   if [[ -n ${fuse_info} ]]; then
     echo -e "pid\tmount_point\talluxio_path"
-    echo -e "$(ps ax -o pid,args | grep [A]lluxioFuse | awk -F' ' '{print $1 "\t" $(NF-2) "\t" $NF}')"
+    echo -e "$(ps ax -o pid,args | grep [A]lluxioFuse | awk -F' ' '{print $1 "\t" $(NF-4) "\t" $NF}')"
```
Comment (yyongycy) [inline integration/fuse/bin/alluxio-fuse:205]: Is the behavior of linux and mac same?

Comment (yyongycy) [reply integration/fuse/bin/alluxio-fuse:205]: Could you please try both and paste the results in this PR?

Comment (secfree) [reply integration/fuse/bin/alluxio-fuse:205]: The demo is from a linux env. I just tested it with my mac, it is the same.
After applying this PR, it became ok.

```
> integration/fuse/bin/alluxio-fuse mount -o allow_other,default_permissions /tmp/projects /projects
Starting AlluxioFuse process: mounting alluxio path "/projects" to local mount point "/tmp/projects"
usage: sleep seconds
Successfully mounted Alluxio to /tmp/projects.

> jps -m
87837 AlluxioFuse -m /tmp/projects -a /projects -o allow_other,default_permissions

> integration/fuse/bin/alluxio-fuse unmount -f /tmp/projects
Successfully unmount fuse at /tmp/projects

> jps -m
87837 AlluxioFuse -m /tmp/projects -a /projects -o allow_other,default_permissions
```

**Review by yyongycy — APPROVED** [review]
LGTM

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**